tests/unit/moduleapi/testmodule.tcl

```diff
@@ -0,0 +1,13 @@
+set testmodule [file normalize src/modules/testmodule.so]
+
+
+# TEST.CTXFLAGS requires RDB to be disabled, so override save file
+start_server {tags {"modules"} overrides {save ""}} {
+    r module load $testmodule
+
+    test {TEST.IT runs successfully} {
+        r test.it
```

Comment:
TEST.IT is not a very good name. please rename to indicate what's being tested

Reply:
ohh, i now realize this utilizes a module from `src/modules` with some preexisting tests (not part of `tests/modules` folder).
i'll rebase this and sort this out.

Reply:
Much of this PR makes more sense when ones notes it was from 2016.    I was trying to improve the at-the-time-brand-new testing modules without changing anything of antirez's  -- especially as an external contributor just trying to get his bugfix accepted.  Thanks again for realizing this and just taking care of it all!

Reply:
yes, understood. judging it in today's context so many years later is odd.
back then, there was barely infrastructure for module testing (and no CI)..